rules/explain-deleted-or-modified-appointments/rule.md

```diff
@@ -22,8 +22,15 @@ guid: 1c278726-9df4-44dc-b09b-bb17470c31ad
 
 ---
 
+Every change to an appointment is a hassle and it's good to be explicit as to why the appointment had to change. There are also people that need some pressure put on them when they change an appointment multiple times so that we don't end up on V7 🤣 (see Version Numbers below).
+
 Whenever you have to modify the time, attendees, or subject of an appointment, always send the updated appointment with comments at the top to explain why you want to make these changes.
 
+There are two key ways to do this:
+
+1. Actions words - These are prepended to any notes that you add to the top of the appointment so that people can "at-a-glance", understand what's going on with the change. E.g. "**Updated**: Changed time from 9:00am to 10:00am"
+2. Version Numbers - Adding these to the notes can help people to track all of the changes and, subtly, remind people that if they're on V5 😖, something else might be going wrong. E.g. "**Updated V8**: Moved from Tuesday 7:00pm to Wednesday 9:00am"
```

Comment:
Hey @SamBWagner 

based on my other comment on line 25... If **v7** is unwanted, we shouldn't use **v8** as an example... maybe change to v3

BTW - I think we always use the version number with lowercase "v"

Reply:
I don't agree. I think having different values for the versioning strengthens the comprehension of the reader as they can infer from the changing numbers that the number is a variable that can change. There is no meaning in any given number, thus changing it conveys that to the reader.